integration/test_gpu_decoder_config.py

```diff
@@ -0,0 +1,102 @@
+"""Test basic functionality relating to the gpu_decoder_config."""
+
+from datetime import datetime
+from pathlib import Path
+
+import pandas as pd
+
+import qnexus as qnx
+from qnexus.models.references import GpuDecoderConfigRef
+
+
+def test_gpu_decoder_config_download(
+    _authenticated_nexus: None,
+    qa_project_name: str,
+) -> None:
+    """Test that valid GPU decoder config can be extracted from an uploaded GPU decoder config."""
+
+    my_proj = qnx.projects.get(name_like=qa_project_name)
+
+    gpu_decoder_config_path = Path(
+        "examples/basics/data/gpu_decoder_config.yaml"
+    ).resolve()
+    with open(gpu_decoder_config_path) as fp:
+        gpu_decoder_config = fp.read()
+
+    qa_gpu_decoder_config_name_fixture = (
+        f"qnexus_integration_test_gpu_decoder_config_{datetime.now()}"
+    )
+
+    qnx.gpu_decoder_configs.upload(
+        gpu_decoder_config=gpu_decoder_config,
+        name=qa_gpu_decoder_config_name_fixture,
+        project=my_proj,
+    )
+
+    gpu_decoder_config_ref = qnx.gpu_decoder_configs.get(
+        name_like=qa_gpu_decoder_config_name_fixture
+    )
+    assert isinstance(gpu_decoder_config_ref, GpuDecoderConfigRef)
+
+    downloaded_gpu_decoder_config = (
+        gpu_decoder_config_ref.download_gpu_decoder_config_contents()
+    )
+
+    assert downloaded_gpu_decoder_config == gpu_decoder_config
+
+
+def test_gpu_decoder_config_flow(
+    _authenticated_nexus: None,
+    qa_project_name: str,
+    qa_qir_name: str,
+) -> None:
+    """Test the flow for executing a simple GPU decoder config circuit on H1-1E."""
+
+    device_name = "H1-1E"
```

Comment:
I think this will need to be a next-gen device, might be worth using the NEXUS_QA_QSYS_DEVICE?

Reply:
have used Helios-1E which I think is acceptable (as per Fernando's other tests)